# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libetpan/0003-fix-date_time_send.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From 5e442095eb236f81a50f61ce0c37079c40c0d501 Mon Sep 17 00:00:00 2001
From: Nadeem Shehzad <Nadeem.Shehzad@securepoint.de>
Date: Tue, 25 Jan 2011 12:14:53 +0100
Subject: [PATCH 3/3] fix: date_time_send by assigning correct value to date_year_send i.e. send rather than month.

---
 src/low-level/imap/mailimap_sender.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/low-level/imap/mailimap_sender.c b/src/low-level/imap/mailimap_sender.c
index 8c9f6e6..3ab5603 100644
--- a/src/low-level/imap/mailimap_sender.c
+++ b/src/low-level/imap/mailimap_sender.c
@@ -907,7 +907,7 @@ mailimap_date_time_send(mailstream * fd,
   if (r != MAILIMAP_NO_ERROR)
     return r;
 
-  r = mailimap_date_year_send(fd, date_time->dt_month);
+  r = mailimap_date_year_send(fd, date_time->dt_year);
   if (r != MAILIMAP_NO_ERROR)
     return r;
 
-- 
1.6.3.3

